src/Aspire.Dashboard/wwwroot/js/app.js

```diff
@@ -370,3 +370,7 @@ window.downloadStreamAsFile = async function (fileName, contentStreamReference)
     anchorElement.remove();
     URL.revokeObjectURL(url);
 };
+
+window.getUserAgent = function() {
```

Comment:
There is getBrowserTimeZone method. This info should be combined with it. It could be called something like `getBrowserInfo`.

Browser details can be called and cached at the same time to avoid many roundtrips to the client.